update status report for December changes

techliaison opened the pull request:
changes in both the FPKIAR main report & the Availability repo page
let me know if there are issues with these updates

techliaison committed:
update status report for December changes

techliaison committed:
Update fpkiar-repo-table.html

techliaison committed:
Update fpkiar-repo-table.html

techliaison committed:
fix shading

techliaison committed:
Update 03_fpki_activity_report.md

lachellel commented on _tools/03_fpki_activity_report.md:
```diff
@@ -30,46 +30,47 @@ The following certificates were issued **BY** or **TO** the FPKI Trust Infrastru
+| USPTO	| Federal Bridge CA 2016	| USPTO_INTR_CA1	| d8d860c11f216ad1fe07e15caf19b3083c83dbba	| 12/19/2019 | Expired |
+| FPKIMA	| SHA-1 Federal Root CA G2	| Federal Common Policy CA	| 27c589ff2853bd1949cfa433f36a5e285b2e2c7c	| 12/31/2019 | Expired |
+| State	| U.S. Department of State AD Root CA	| Federal Common Policy CA	| c8feb56d0efa3b3b4e2777986304cc19118676a4	| 1/27/2020 | Renewed |
+| Treasury	| US Treasury Root CA	| Federal Common Policy CA	| 5a87922b5eaf1d63198a951b2ab6f59b2f16c131	| 8/29/2021 | Renewed |
```
@ryancdickson does this (treasury) belong in the "removed" in the last 30 days section?

lachellel committed:
Updates for fpkiar

Do these updates make sense (removing Treasury and State from the removal table)? 
I also suggest / made updates to the language under repository availability.

techliaison reviewed (commented):
why did you remove the state certificate?
This one was removed from the FCPCA's SIA p7c and replaced with the latest cert issued to State.
